maintainers/maintainer-list.nix

```diff
@@ -1707,6 +1707,12 @@
     githubId = 3086255;
     name = "Barry Moore II";
   };
+  chivay = {
```

Comment:
This change should go into a separate commit in this PR.

Reply:
Fixed